.github/workflows/ci.yml

```diff
@@ -212,9 +255,19 @@ jobs:
     outputs:
       tag: ${{ steps.version.outputs.version }}
 
-  publish-nuget-pre-release:
+  # Virtual job that can be configured as a required check before a PR can be
+  # merged.
+  all-required-checks-done:
+    needs: [check-style, build, test, package]
+    runs-on: ubuntu-latest
+    steps:
+      - run: echo "All required checks done"
+
+  # Publish a preview version of the NuGet packages on feedz.io when there is
+  # a push to master.
+  publish-preview:
     if: github.ref == 'refs/heads/master' && github.event.repository.fork == 'false'
```

Comment:
QUESTION: I think this `if` condition will cause the scheduled job to publish a preview nuget package every night. Should we disable it for scheduled jobs?

Reply:
Agreed, thanks for catching it @MoFtZ!

Reply:
Fixed in https://github.com/m4rs-mt/ILGPU/pull/536/commits/da40b7195110fa6f16604548477ba32555ce0503